Backend/Underdogs-Struggling/src/main/java/com/TheTroisMousquetaires/Underdogs/Struggling/entities/Language.java

```diff
@@ -0,0 +1,17 @@
+package com.TheTroisMousquetaires.Underdogs.Struggling.entities;
+
+import jakarta.persistence.*;
+import java.util.*;
+
+@Entity
+@Table(name = "languages")
+public class Language {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "language_id")
+    private Long languageID;
+
+    @Column(name = "language_code", unique = true, nullable = false)
+    private String languageCode;
+    
+}
```

Comment:
Should we add one more column `language`. For example we added English as first language, id is 1, code is EN, and `language` field will be "English".

Reply:
Yes, you are correct. I will add a `languageName` attribute to make sure there will not be any mapping design in the front end.
And for the attribute `languageCode` will be retrieved and create a Locale class based on this code.

Reply:
Solved